lib/web/fotorama/fotorama.js

```diff
@@ -1140,7 +1153,13 @@ fotoramaVersion = '4.6.4';
 
     function addEvent(el, e, fn, bool) {
         if (!e) return;
-        el.addEventListener ? el.addEventListener(e, fn, !!bool) : el.attachEvent('on' + e, fn);
+
+        const options = (Modernizr.passiveeventlisteners) ? {
```

Comment:
just curiously do we actual need add passive for every dom element ? CC: @fredden 
Can we add logic for check specific

Reply:
element window.document or window.document.body or window should be set passive false

Reply:
Looking at [this page on `developer.mozilla.org`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#improving_scrolling_performance_with_passive_listeners), it seems that omitting 'passive' seems to be the best option as modern browsers\* default the 'passive' option to a sensible value depending on the event.

\* but not Safari. Safari seems to often lag when implementing features. For example, the 'loading=lazy' attribute for '&lt;img&gt;'.

Reply:
yeap i think default passive enabled on safari and it's supported well
https://caniuse.com/passive-event-listener
only not support for retired IE browser